metadb/scrapers/musicbrainz.py

```diff
@@ -0,0 +1,33 @@
+import getpass as gt
+import sys
+import musicbrainzngs as mb
+
+username = raw_input('Enter MusicBrainz Username: ')
+password = gt.getpass('Password for {}: '.format(username))
+mb.auth(username,password)
```

Comment:
we don't need a username and password here - these are only needed for queries where you add the `include` value `user-tags`, that is, tags that this specific user has added to the recording. You can remove this